core/config.lua

```diff
@@ -83,4 +89,4 @@ TC2.defaultConfig.warnings = {
 	threshold			= 80,									-- alert threshold (of normalized percentage 0-100)
 	warningFile			= [[Sound\Interface\Aggro_Enter_Warning_State.ogg]],
 	pulledFile			= [[Sound\Interface\Aggro_Pulled_Aggro.ogg]],
-}
+}
```

Comment:
please keep the newlines at end of file for consistency

Reply:
Readded